brewtils/pika.py

```diff
@@ -545,11 +545,9 @@ def on_connection_closed(self, connection, *args):
         Args:
             connection: The connection
             args: Tuple of arguments describing why the connection closed
-                pika < 1:
-                    reply_code: Numeric code indicating close reason
-                    reply_text: String describing close reason
-                pika >= 1:
-                    exc: Exception describing close
+                For pika < 1: reply_code (Numeric code indicating close reason),
```

Comment:
The previous text rendered in pretty broken fashion.  I couldn't come up with anything that rendered nicely, so instead I settled for at least not looking broken.